plugin/trino-influxdb/src/main/java/io/trino/plugin/influxdb/InfluxDataType.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.trino.plugin.influxdb;
+
+import io.trino.spi.type.BigintType;
+import io.trino.spi.type.BooleanType;
+import io.trino.spi.type.DoubleType;
+import io.trino.spi.type.Type;
+import io.trino.spi.type.VarcharType;
+
+import java.util.Optional;
+
+public final class InfluxDataType
+{
+    public static final String STRING = "string";
+    public static final String BOOLEAN = "boolean";
+    public static final String INTEGER = "integer";
+    public static final String FLOAT = "float";
+    public static final String TIMESTAMP = "timestamp";
+
+    private InfluxDataType() {}
+
+    public static Optional<Type> toTrinoType(String influxType)
+    {
+        Type type = switch (influxType) {
+            case STRING -> VarcharType.VARCHAR;
+            case BOOLEAN -> BooleanType.BOOLEAN;
+            case INTEGER, TIMESTAMP -> BigintType.BIGINT;
```

Comment:
Why dont we want to map  InfluxDB`Timestamp` to Trino's `Timestamp` ?

Reply:
can map to TIMESTAMP_NANOS?
https://docs.influxdata.com/influxdb/v1.8/write_protocols/line_protocol_reference/#data-types